src/analysis/modals/RuntimeErrorModal.ts

```diff
@@ -0,0 +1,79 @@
+import { Modal, ModalProps } from '@terra-ui-packages/components';
+import { useNotificationsFromContext } from '@terra-ui-packages/notifications';
+import _ from 'lodash/fp';
+import { ReactNode, useState } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { spinnerOverlay } from 'src/components/common';
+import { LeoRuntimeProvider, RuntimeBasics } from 'src/libs/ajax/leonardo/providers/LeoRuntimeProvider';
+import colors from 'src/libs/colors';
+import { useOnMount } from 'src/libs/react-utils';
+import { withBusyState } from 'src/libs/utils';
+
+export type RuntimeErrorProvider = Pick<LeoRuntimeProvider, 'errorInfo'>;
+
+export const text = {
+  error: {
+    title: {
+      standard: 'Cloud Environment is in error state',
+      userScript: 'Cloud Environment is in error state due to Userscript Error',
+    },
+    unknown: 'An unknown error has occurred with the runtime',
+    cantRetrieve: 'Could Not Retrieve Cloud Environment Error Info',
```

Comment:
Why is this one not in sentence case like the others?

Reply:
the strings are materially unchanged from what was in the original RuntimeErrorProvider code in AnalysisNotificationManager.ts